src/spurt/workflows/emcf/_settings.py

```diff
@@ -12,9 +12,13 @@ class SolverSettings:
 
     Parameters
     ----------
-    worker_count: int
+    t_worker_count: int
```

Comment:
Note that this change (renaming `worker_count` to `t_worker_count`) breaks the CLI and one of the unit tests

https://github.com/isce-framework/spurt/blob/b48bfb2dc452bfd38141160f0c6b4154ee710607/src/spurt/workflows/emcf/_cli.py#L82-L83

https://github.com/isce-framework/spurt/blob/b48bfb2dc452bfd38141160f0c6b4154ee710607/test/workflows/test_emcf.py#L88